installer/create-cluster-aks.sh

```diff
@@ -113,9 +118,12 @@ az aks create --resource-group "$RESOURCE_GROUP_NAME" \
   --name "$CLUSTER_NAME" \
   --node-count $NODE_COUNT \
   --enable-addons monitoring \
+  --enable-cluster-autoscaler \
   --generate-ssh-keys \
   --kubernetes-version "$CLUSTER_VERSION" \
-  --node-vm-size $NODE_VM_SIZE
+  --node-vm-size $NODE_VM_SIZE \
+  --min-count 1 \
+  --max-count 7
```

Comment:
How are min/max count related to `$NODE_COUNT` ? 
e.g. if the user gives a $NODE_COUNT > max-count, what's the expected behavior?

Reply:
Good question. I haven't tested that scenario yet. Our GKE cluster creation script has the same issue. I'll update both scripts to check that node count is between and min and max counts.

Reply:
In the script for GKE, we hard code the min, max and default node counts to be 1, 7 and 5, respectively. I updated this script to do the same. The user can go in the script and change the numbers if they want.